src/controllers/controller.doughnut.js

```diff
@@ -322,10 +343,11 @@ class DoughnutController extends DatasetController {
 	}
 
 	/**
+	 * @param {number} datasetIndex
```

Comment:
optional: you could also add `@return {number}` here and a return value on the other methods if you want to document things ;-)